app/cdap/components/WranglerV2/SearchResultsListItem/index.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import { Box } from '@material-ui/core';
+import styled from 'styled-components';
+import ArrowForwardIosOutlinedIcon from '@material-ui/icons/ArrowForwardIosOutlined';
+
+export interface ISearchResultListItemProps {
+  primaryText: string;
+  secondaryText: string;
+  icon: boolean;
+  onClick: () => void;
+}
+
+const SearchBarWrapper = styled(Box)`
+  display: flex;
+  margin: 0px 20px;
+  padding: 15px;
+  align-items: flex-end;
+  cursor: pointer;
+  border-bottom: 1px solid #e0e0e0;
+  '& :hover': {
+    backgroundColor: #EFF0F2,
```

Comment:
use mui colors

Reply:
above color is not available in MUI colors